src/util/triplea/MapXMLCreator/MapXMLCreator.java

```diff
@@ -0,0 +1,839 @@
+		m_bAuto.setMargin(new Insets(2, 5, 2, 5));
+		m_bAuto.setMinimumSize(new Dimension(50, 23));
+		m_bAuto.setMaximumSize(new Dimension(50, 23));
+		southCenterPanel.add(m_bAuto);
+		
+		m_bNext = new JButton("Next");
+		m_bNext.setMnemonic(KeyEvent.VK_N);
+		m_bNext.setFont(new Font("Tahoma", Font.PLAIN, 11));
+		m_bNext.addActionListener(new AbstractAction("Next Step")
+		{
+			private static final long serialVersionUID = 6322566373692205163L;
+
+			public void actionPerformed(final ActionEvent e)
+			{
+				SwingUtilities.invokeLater(new Runnable()
+				{
+					public void run()
+					{
+						if (m_currentStep < m_stepList.size())
+						{
+							goToStep(m_currentStep+1);
+							m_actionPanel.validate();
+							m_actionPanel.repaint();
+						}
+					}
+				});
+			}
+		});
+		southCenterPanel.add(m_bNext);
+		
+		southRightPanel = new JPanel();
+		FlowLayout fl_southRightPanel = (FlowLayout) southRightPanel.getLayout();
+		fl_southRightPanel.setAlignment(FlowLayout.RIGHT);
+		m_southPanel.add(southRightPanel);
+		
+		m_bAvailableChoices = new JButton("Available Choices");
+		m_bAvailableChoices.setMnemonic('C');
+		m_bAvailableChoices.setMargin(new Insets(2, 5, 2, 5));
+		m_bAvailableChoices.setFont(new Font("Tahoma", Font.PLAIN, 11));
+		m_bAvailableChoices.addActionListener(new ActionListener()
+		{
+			public void actionPerformed(ActionEvent e)
+			{
+				switch (m_currentStep)
+				{
+					case 1: // showInfoMessage("Map Name: The map name that is displayed in the New Game window in TripleA. Examples: Revised, Classic, Big World, and Great War\r\n\r\nMap Version: The version of the map's xml file. Examples: 0.0.1, 1.0.0.1, and 1.1\r\n\r\nResource Name: The name of the resource used in the map. Resources can be thought of as buying tokens or currency used to buy units, technology, etc. 'IPCs' have recently been replaced by 'PUs'. 'IPCs' will no longer work as the map resource. \r\n\r\nMap Image Location: The location of the map image. Example: C:/My Maps/Sleeping Giant/full_map.png\r\n\r\nMap Centers File: The location of the centers file produced by the 'Center Picker' program. The centers file is used to automatically add the map's territories. Example: C:/My Maps/Sleeping Giant/centers.txt\r\n\r\nWater Territory Filter: An optional setting that makes the program automatically apply the 'Is Water' property to every territory that contains the filter text. Examples: SZ, Sea Zone, Pacific, and Atlantic.", "Help For Current Step");
+						break;
+					case 2: // showInfoMessage("To add a new territory, click somewhere on the map and enter a name for the territory in the window that appears. \r\n If you want to change the properties of a territory, left click on it and answer each question. The color of the territory label changes for each property that is applies. If you want to remove a territory label, right click on it and click yes when it asks for confirmation.", "Help For Current Step");

```

Comment:
I will add a TODO to check the message text that should be displayed here.
Basically I was taking over message text from the C++ implementation and didn't have time to review everything myself.